src/qml/qgismobileapp.qml

```diff
@@ -192,7 +192,11 @@ ApplicationWindow {
 
     HoverHandler {
         id: hoverHandler
-        enabled: !qfieldSettings.mouseAsTouchScreen && !gpsLinkButton.linkActive && !parent.isBeingTouched && !digitizingToolbar.rubberbandModel.frozen
+        enabled: !qfieldSettings.mouseAsTouchScreen
+                 && !gpsLinkButton.linkActive
+                 && !parent.isBeingTouched
+                 && digitizingToolbar.rubberbandModel
```

Comment:
@nirvn I am not 100% sure what is the correct behaviour when the `rubberbandModel` is not present, it might need the logic here to be inverted?

Reply:
To be safe, I'd do ... && (!rubberband || !rubberband.frozen)

Reply:
Thanks